### ISSUE TITLE
AllSky Admin Panel Questions

### ISSUE THREAD

**Issue by maserowik** [thomasjacquin/allsky-portal#81]
I am color blind and having a issue with one of the line in the system information.
The Throttle Status is I believe red on green.  

![image](https://user-images.githubusercontent.com/25713782/146989425-1435e661-8c3d-4ed9-a672-ad38f0b9987b.png)

anyway to drop the red and make it like the others?

![image](https://user-images.githubusercontent.com/25713782/146989483-6706ab90-4ae3-447b-9e1c-490fd191c5aa.png)

Thanks
Mike

**Comment by EricClaeys** [comment]
@maserowik I just submitted a fix for this.  The font color for the throttle status bar was the same as the characters to the left of the green bars, which is gray.  The other status bars have white text.
While fixing this I noticed in the code that the bars are supposed to grow from 0 to the correct width in 1/2 second when the page is loaded, so you see them moving.  My HTML and CSS skills aren't good enough to fix it, but it has my interest so I created Issue #83.
